wiiu/vino/vino.ts

```diff
@@ -1003,38 +1090,38 @@ export default interface VinoAPI {
 	title_setAnimImageEx: (unk1: string, unk2: string, unk3: string, unk4: string, unk5: string, unk6: number) => boolean;
 
 	/**
-	 * Unknown
-	 * @param unk1 - Unknown
-	 * @param unk2 - Unknown
+	 * Sets an image in the title screen
+	 * @param URL - PNG image URL
```

Comment:
You've used `URL` before to seemingly only mean parts of a URL. IS this a fully qualified URL? Also do not fully caps the parameter names